ui-components/src/headers/StepHeader.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from "react"
+import { t } from "../.."
+import "./StepHeader.scss"
+
+export interface StepHeaderProps {
+  currentStep: number
+  totalSteps: number
+  stepTitle?: string
+}
+
+const StepHeader = ({
+  currentStep,
+  totalSteps,
+  stepTitle = t("stepHeader.defaultLabel"),
```

Comment:
We can't add any references to `t` in ui-components because of consumers who don't use translations this way. We'll have to pass the default as we consume the component.